app/assets/javascripts/csv_field_checker.js

```diff
@@ -304,11 +326,16 @@ function fillInTableFromFile() {
             // A-1 format
             wellPosition = wellPosition.split('')
             wellPosition.splice(1, 1, ':')
+          } else if (/^([a-zA-Z]([0-9]+))$/.test(wellPosition)) {
```

Comment:
I think this is dead code as it's the same check that's carried out in the first 'if' block